### PR TITLE
Fix, test and deprecate petab.sbml.constant_species_to_parameters

### PR DESCRIPTION
**Pull request by dweindl**
Function failed in case of multiple occurrences of the same species.

### CODE REVIEW

File: petab/sbml.py [hunk 1]
```diff
@@ -106,9 +111,12 @@ def constant_species_to_parameters(sbml_model: libsbml.Model) -> list:
     # Remove from reactants and products
     for reaction in sbml_model.getListOfReactions():
         for speciesId in transformables:
-            reaction.removeReactant(speciesId)
-            reaction.removeProduct(speciesId)
-            reaction.removeModifier(speciesId)
+            while reaction.removeReactant(speciesId):
```
Comment (yannikschaelte) [inline petab/sbml.py]: add comment that the loop is because the functions always only remove 1? helps code maintenance later.

Comment (dweindl) [reply petab/sbml.py]: done

**Comment by paulstapor** [comment]
I updated your branch, Daniel. Now the checks pass.
Feel free to merge whenever you want... 🙂

**Comment by dweindl** [comment]
Thanks @paulstapor

**Pull request merged**